Elements.css

```diff
@@ -1,117 +1,141 @@
 #outside_container {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: url(background.jpg);
-  z-index: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(background.jpg);
+    z-index: -1;
 }
 
 #container {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    margin-bottom: 50px;
+    overflow: auto;
 }
 
 .container_2 {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    border: 2px solid #000000c5;
```

Comment:
It's ok to use extended HEX colours with opacity, but keep it in mind, that will not be supported by the older browsers.

Reply:
fixed